src/app/components/system-maintenance/api-key/api-keys-view.jsx

```diff
@@ -67,42 +74,41 @@ function ChplApiKeysView({ apiKeys: initialApiKeys, dispatch }) {
           title="API Keys History"
         />
       </div>
-      <TableContainer component={Paper}>
-        <Table
-          aria-label="API Keys table"
-        >
-          <ChplSortableHeaders
-            headers={headers}
-            onTableSort={handleTableSort}
-            orderBy={orderBy}
-            order={order}
-            stickyHeader
-          />
-          <TableBody>
-            { apiKeys
-              .map((key) => (
+      <Paper className={classes.container}>
+        <TableContainer>
+          <Table aria-label="API Keys table">
+            <ChplSortableHeaders
+              headers={headers}
+              onTableSort={handleTableSort}
+              orderBy={orderBy}
+              order={order}
+              stickyHeader
+            />
+            <TableBody>
+              {apiKeys.map((key) => (
                 <TableRow key={key.key}>
-                  <TableCell className={classes.firstColumn}>{ key.name }</TableCell>
-                  <TableCell>{ key.email }</TableCell>
-                  <TableCell>{ key.key }</TableCell>
-                  <TableCell>{ getDisplayDateFormat(key.lastUsedDate) }</TableCell>
-                  <TableCell>{ getDisplayDateFormat(key.deleteWarningSentDate) }</TableCell>
-                  <TableCell align="right">
+                  <TableCell className={classes.firstColumn}> {key.name} </TableCell>
```

Comment:
```suggestion
                  <TableCell className={classes.firstColumn}>{ key.name }</TableCell>
```

You put the spaces in the wrong place